code/game/objects/items/weapons/grenades/ghettobomb.dm

```diff
@@ -2,7 +2,7 @@
 
 /obj/item/grenade/iedcasing
 	name = "improvised firebomb"
-	desc = "A weak, improvised incendiary device."
+	desc = "An improvised incendiary device of dubious quality."
```

Comment:
improvised and dubious quality is kinda saying the same thing, could do without repeating itself.

Reply:
It is now sketchy.